test/export/test_export.py

```diff
@@ -5382,6 +5382,25 @@ def forward(self, w, x, y, z):
                 _disable_forced_specializations=True,
             )
 
+    @testing.expectedFailureSerDer
```

Comment:
What fails here? Could you file issue?

Reply:
I think requires_grad are not preserved with roundtripping. I can file an issue later.

Reply:
Can you leave a message describing what doesn't work then?